packages/sources/amberdata/src/endpoint/crypto.ts

```diff
@@ -0,0 +1,64 @@
+import { Requester, Validator } from '@chainlink/ea-bootstrap'
+import { ExecuteWithConfig, Config } from '@chainlink/types'
+import { NAME as AdapterName } from '../config'
+
+export const Name = 'crypto'
```

Comment:
We will want to maintain `price` here, otherwise this might be a breaking change for some people.

Reply:
Yeah, this will be part of PR but the new update to the adapter endpoint handling it will be easy to support both the endpoints 'price' and 'crypto' for one 'endpoint'